app/views/users/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="container-fluid">
+  <section id="player-dashboard">
+    <div class="row">
+      <div class="user col-sm-offset-1 col-sm-2">
+        <%= image_tag 'user.png', class:'img-responsive' %>
+        <h4><%= current_user.email %></h4>
+        <h4>Member Since: <%= current_user.created_at.strftime("%B %d, %Y") %></h4>
+        <a class="btn-block btn btn-info account-settings" href="#" role="button">Account Settings</a>
+      </div>
+      <div class="boxed col-sm-8 col-xs-10 col-sm-offset-0 col-xs-offset-1">
```

Comment:
From the documentation - "You can also override offsets from lower grid tiers with .col-*-offset-0 classes." - However I'm not sure I'm using it correctly? I will go over it regardless